src/radical/saga/engine/engine.py

```diff
@@ -251,7 +252,8 @@ def _load_adaptors (self, inject_registry=None):
             adaptor_enabled = False
 
             try :
-                adaptor_config  = ru.Config('radical.saga', name=adaptor_name)
+                adaptor_config  = ru.Config('radical.saga.adaptors',
+                                            name=adaptor_name)
```

Comment:
`'radical.saga.adaptors'` is the module right? May I suggest to change it to:
```python
adaptor_config = ru.Config(module='radical.saga.adaptor',
                                                      name=adaptor_name)
```

It creates a redundancy, but I think it makes more consistent with lines 132-134

Reply:
I removed the `module=` part in 132-134, if that's ok for you?  Rationale being that `module` is actually not a good name, it's more a namespace, really, and we may want to change that on RU level (out of scope for this PR though).  Does that work?

Reply:
Yes!